### ISSUE TITLE
Move Scapix to a separate CMake file

### ISSUE THREAD

**Issue by TimPushkin** [TimPushkin/SgbmDepthAndroid#8]
Currently Scapix version from `cmodule-1.0.32` is used. Because of a bug in this version, Scapix has to be configured in the same CMake file where the library target is created, and examples (and any other targets using the library) cannot be build if Scapix bridge is used.

In version from `cmodule-1.0.34` the bug is fixed, so Scapix can be moved to a separate file and be used along side with examples. But in `cmodule-1.0.33` another problem was introduced: OpenCV build fails with the following errors:
```
In file included from < project path >/SgbmAndroid/sgbmandroid/src/main/cpp/include/DepthEstimator.h:7:
In file included from < project path >/SgbmAndroid/sgbmandroid/src/main/cpp/build/external/opencv/install/include/opencv4/opencv2/core.hpp:52:
In file included from < project path >/SgbmAndroid/sgbmandroid/src/main/cpp/build/external/opencv/install/include/opencv4/opencv2/core/cvdef.h:349:
In file included from < project path >/SgbmAndroid/sgbmandroid/src/main/cpp/build/external/opencv/install/include/opencv4/opencv2/core/cv_cpu_dispatch.h:210:
/usr/lib/llvm-10/lib/clang/10.0.0/include/emmintrin.h:2374:19: error: use of undeclared identifier '__builtin_ia32_pmaxsw128'
  return (__m128i)__builtin_ia32_pmaxsw128((__v8hi)__a, (__v8hi)__b);
                  ^
/usr/lib/llvm-10/lib/clang/10.0.0/include/emmintrin.h:2394:19: error: use of undeclared identifier '__builtin_ia32_pmaxub128'
  return (__m128i)__builtin_ia32_pmaxub128((__v16qi)__a, (__v16qi)__b);
                  ^
/usr/lib/llvm-10/lib/clang/10.0.0/include/emmintrin.h:2414:19: error: use of undeclared identifier '__builtin_ia32_pminsw128'
  return (__m128i)__builtin_ia32_pminsw128((__v8hi)__a, (__v8hi)__b);
                  ^
/usr/lib/llvm-10/lib/clang/10.0.0/include/emmintrin.h:2434:19: error: use of undeclared identifier '__builtin_ia32_pminub128'
  return (__m128i)__builtin_ia32_pminub128((__v16qi)__a, (__v16qi)__b);
                  ^
```

The work on fixing this is being done in `scapix` branch.

**Comment by TimPushkin** [comment]
I'm also experiencing a strange behaviour of `ExternalProject_Add` which seems to download and build OpenCV after Scapix, which results in errors because the linker cannot find OpenCV included in `DepthEstimator.h` when using `cmodule-1.0.34`. But I'm not sure if this is not just some random local error.

**Comment by TimPushkin** [comment]
Turned out that `clang 14` or newer is required for newer Scapix to build with OpenCV (maybe `clang 13` is also acceptable, but `12` and older are definitely not)

**Comment by TimPushkin** [comment]
Fixed in #13
